finetrainers/args.py

```diff
@@ -0,0 +1,779 @@
+import argparse
+from typing import Any, Dict, List, Optional, Tuple
+
+from .constants import DEFAULT_IMAGE_RESOLUTION_BUCKETS, DEFAULT_VIDEO_RESOLUTION_BUCKETS
+
+
+class Args:
```

Comment:
This could be managed with `dataclass`. Examples:

* https://github.com/huggingface/trl/blob/0fe73a8af5ff660becc79bff88d9e8b090dd004f/trl/trainer/alignprop_config.py#L26